client/src/components/ActivityList/ActivityList.js

```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import "./ActivityList.css";
+
+// Class Based React Component
+class ActivityList extends Component{
```

Comment:
Both `Activity` and `ActivityList` should be function components, there wont be any state that either needs to manage.